src/huak/ops/activate.rs

```diff
@@ -0,0 +1,15 @@
+use crate::{errors::HuakResult, project::Project};
+
+pub fn activate_project_venv(project: &Project) -> HuakResult<()> {
+    let venv = project
+        .venv()
+        .as_ref()
+        .expect("`Project::from` creates venv if it doesn't exists.");
```

Comment:
Should we handle this using `HuakError`?
```suggestion
    let venv = match project.venv() {
        Some(v) => v,
        _ => return Err(HuakError::VenvNotFound),
    };
```

Reply:
You're right, after that small refactor we can't be sure that venv is there. 
We could: 
1. Take venv instead of project, but then that function is just printing around the other function (worst option imo)
2. Error as you suggested
3. Create venv if it doesn't exist

Personally I'm in favor of option 3.

Reply:
Yea 3 is nice if we have clear behavior paths for venv usage. This is part of the whole venv-as-a-context thing.

If a project is operated on within a venv context then create a venv.

Or at least this is where my mind has been.

Reply:
I'm slightly confused, because you merged this PR without this being fixed. Are you planning to fix this later, once venv situation has improved?

Reply:
I didn't have the windows build action step in CI. So it would have caught it if I did. That's on me.

Reply:
Saw this from email. Didn't realize it was the error handling. I see. Didn't notice this wasn't complete.

Reply:
Should I fix this and make a PR?

Reply:
Would love that. Was going to sneak this into that PR I was helping with, but git reset and forgot to re-add.

Reply:
Hense the premature resolution here. heh